app/app/settings/base.py

```diff
@@ -129,28 +130,16 @@
 
 
 """ DATABASE CONFIGURATION """
-if os.getenv("GITHUB_WORKFLOW"):
```

Comment:
github actions uses its own environment variables for building, keep this if-check to make sure it builds correctly